packages/@aws-cdk/aws-codedeploy/README.md

```diff
@@ -43,6 +43,37 @@ const deploymentGroup = codedeploy.ServerDeploymentGroupRef.import(this, 'Existi
 });
 ```
 
+### Deployment Configurations
+
+You can also pass a Deployment Configuration when creating the Deployment Group:
+
+```ts
+const deploymentGroup = new codedeploy.ServerDeploymentGroup(this, 'CodeDeployDeploymentGroup', {
+    deploymentConfig: codedeploy.ServerDeploymentConfig.AllAtOnce,
+});
+```
+
+The default Deployment Configuration is `ServerDeploymentConfig.OneAtATime`.
+
+You can also create a custom Deployment Configuration:
```

Comment:
Maybe in a subsequent commit, but can we provide a sugar that allows me to do this:

```ts 
deploymentConfig: ServerDeploymentConfig.custom({ minHealthyCount: 5 })
```

It will be the natural thing for people to look for when they are “browsing” the options.

The challenge would be how to propagate the parent construct. I think that the construct id can be just DeploymentConfig if you attach this to the DeploymentGroup because a deployment group can only have one deployment config.

Reply:
Yeah, I don't see an easy way of allowing both that form, and `deploymentConfig: new ServerDeploymentConfig(this, 'id', { minHealthyHostCount: 5 })`...